File: Leetcode/sbgo/Leetcode1446-连续字符串.go
package sbgo

func maxPower(s string) int {
	res := 0
	for i, j := 0, 0; i < len(s); i++ {
		for j < len(s) && s[i] == s[j] {
			j++
		}
		res = max(res, j-i)
		i = j - 1
	}
	return res
}

File: Leetcode/JS/Leetcode1446-连续字符串.js
var maxPower = function(s) {
    let res = 0;
    for(let i = 0,j = 0;i < s.length;i++){
        while(j < s.length && s[i] == s[j]) j++;
        res = Math.max(res,j - i);
        i = j - 1;
    }
    return res;
};